### PR TITLE
Patch/code review

### PR DESCRIPTION
**Pull request by jetzlstorfer**

### CODE REVIEW

File: deploy/service.yaml [hunk 1]
```diff
@@ -17,15 +17,15 @@ spec:
     spec:
       containers:
         - name: ansibletower-service
-          image: gabrielprioli/ansibletower-service:latest # Todo: Replace this with your image name
+          image: jetzlstorfer/ansibletower-service:dev # Todo: Replace this with your image name
```
Comment (gabrielprioli) [inline deploy/service.yaml]: @jetzlstorfer do you want to reference the image in your account here?

File: eventhandlers.go [hunk 2]
```diff
@@ -143,7 +147,11 @@ func HandleActionTriggeredEvent(myKeptn *keptn.Keptn, incomingEvent cloudevents.
 		actionResult.Result = "pass"
 		actionResult.Status = "succeeded"
 
-		myKeptn.SendActionFinishedEvent(&incomingEvent, actionResult, data.Labels, incomingEvent.Source())
+		myKeptn.SendActionFinishedEvent(&incomingEvent, actionResult, data.Labels, "ansibletower-service")
+		if err != nil {
+			log.Printf("Got Error From SendActionFinished: %s", err.Error())
```
Comment (gabrielprioli) [inline eventhandlers.go]:
```suggestion
			log.Printf("Got Error From SendActionFinishedEvent: %s", err.Error())
```
Just to keep the standard, I added "Event" here.

**Review by gabrielprioli — CHANGES_REQUESTED** [review]

File: deploy/service.yaml (hunk 1 above)
Comment (jetzlstorfer) [reply deploy/service.yaml]: That was a bit by accident, will change it to `keptnsandbox` as we want to push the images there

Comment (gabrielprioli) [reply deploy/service.yaml]: ok

Comment (jetzlstorfer) [reply deploy/service.yaml]: fixed now

File: eventhandlers.go (hunk 2 above)
Comment (jetzlstorfer) [reply eventhandlers.go]: good catch - fixed it now

**Comment by jetzlstorfer** [comment]
I've updated the README.md but I'm aksing @gabrielprioli to take a look and further improve it if needed.

File: README.md [hunk 3]
```diff
@@ -1,152 +1,185 @@
...
+                JobTemplate: "9"
+    ```
+
+1. Add remediation file to keptn.
+
+    ```
+    keptn add-resource --project=sockshop --stage=production --service=carts --resource=remediation-tower.yaml --resourceUri=remediation.yaml
+    ```
+
+    Note: the name of the remediation file must be `remediation.yaml`.
```
Comment (gabrielprioli) [inline README.md]: @jetzlstorfer the way you rewrote using `remediation-tower.yaml` as file name was much better but now the Note here is confusing. How about we change it to: 
```suggestion
    Note: it is mandatory that the `resourceUri` parameter be exactly `remediation.yaml`
```

File: README.md [hunk 4]
```diff
@@ -1,152 +1,185 @@
...
 
-`keptn send event --file=./(your test file).json`
+    ```
+    keptn send event --file=./(your test file).json
+    ```
 
     Note: change the value between parentesis to match the file name you used for the dummy event.
 
-# Validate
+1.  Optionally you can validate the the logs of the `ansibletower-service` pod.
```
Comment (gabrielprioli) [inline README.md]:
```suggestion
1.  Optionally you can validate the logs of the `ansibletower-service` pod.
```

**Pull request merged**